package.json

```diff
@@ -203,6 +198,11 @@
                 "title": "%containerApps.openConsoleInPortal%",
                 "category": "Azure Container Apps"
             },
+            {
+                "command": "containerApps.updateContainer",
+                "title": "%containerApps.updateContainer%",
+                "category": "Azure Container Apps"
+            },
```

Comment:
Don't want to block this PR from merging, but I think we need to discuss the possibility of rewording the command to make it more clear to users what's happening.

We need users to understand that there are pretty much two main actions for container apps:
1. Editing their Container App configuration (scale rules, etc.) - currently this is "Edit Container App..."
2. Updating the image running in their containers (within the Container App) - currently this is "Update Container..."

Nathan and I think that "Update Container..." might be too close to "Edit Container App...". What about using Deploy instead of Update?

Whatever we decide to do, we can make that change after this is merged.

Reply:
Yup, I agree, and this is something I feel is super important to address.  Unfortunately, as of the time of this PR, we are still transitioning to the ideal end state.  I will try to remember to bring this one up in our next group meeting because I think it's a bit difficult to explain over a comment.

Reply:
I agree, I'll try to squeeze this discussion into one of our next group meetings (probably just the engineers for now). I think these sorts of improvements overlap a ton with my big goal for the next release, and I have some extra context to provide that would also be helpful for this.